### PR TITLE
mopidy-jellyfin: 1.0.2 -> 1.0.4

### PR DESCRIPTION
**Pull request by kiberOgur4ik**
###### Description of changes
https://github.com/jellyfin/mopidy-jellyfin/blob/master/CHANGELOG.rst#v104

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/mopidy/jellyfin.nix [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 python3Packages.buildPythonApplication rec {
   pname = "mopidy-jellyfin";
-  version = "1.0.2";
+  version = "1.0.4";
 
   src = python3Packages.fetchPypi {
     inherit version;
     pname = "Mopidy-Jellyfin";
-    sha256 = "0j7v5xx3c401r5dw1sqm1n2263chjga1d3ml85rg79hjhhhacy75";
+    sha256 = "ny0u6HdOlZCsmIzZuQ1rql+bvHU3xkh8IdwhJVHNH9c=";
```
Comment (lilyinstarlight) [inline pkgs/applications/audio/mopidy/jellyfin.nix:10]: Was there a reason to remove the `sha256-` part of the SRI hash? I'm surprised putting just the base64 part works tbh

File: pkgs/applications/audio/mopidy/jellyfin.nix [hunk 2]
```diff
@@ -23,3 +23,4 @@ python3Packages.buildPythonApplication rec {
     maintainers = [ maintainers.pstn ];
   };
 }
+
```
Comment (lilyinstarlight) [inline pkgs/applications/audio/mopidy/jellyfin.nix:26]: Could you amend your commit without the spurious newline change at the end of the file?

(e.g. remove it and do `git commit -a --amend --no-edit` and `git push --force-with-lease` from your branch)

**Review by lilyinstarlight — COMMENTED** [review]
Welcome to nixpkgs! I left a few minor review nitpicks, but otherwise this PR builds and seems fine to me

File: pkgs/applications/audio/mopidy/jellyfin.nix (hunk 2 above)
Comment (kiberOgur4ik) [reply pkgs/applications/audio/mopidy/jellyfin.nix]: I get an editorconfig error without this newline

Comment (lilyinstarlight) [reply pkgs/applications/audio/mopidy/jellyfin.nix]: You had two newlines and it just needed to be one newline. Right now it's zero newlines (see the red circle thing in the diff). Does that make sense?

Comment (kiberOgur4ik) [reply pkgs/applications/audio/mopidy/jellyfin.nix]: got it, thanks. I seem to have fixed it

**Review by lilyinstarlight — APPROVED** [review]
This looks good to me now!

**Pull request merged**